File: 01-FizzBuzz/Answers/Python/fizzbuzz2.py
#!usr/bin/python
# -*- coding: UTF-8 -*-

# fizzbuzz2.py
# By kota ninomiya

def fizz_buzz(number:int) -> str:
    """fizzbuzzを判断し，その結果を返す．該当しない場合は数字を返す
    
    Args:
        number (int): 検証する値
    
    Returns:
        str: Fizz, Buzz, Fizz Buzz or numer
    """
    if number%15 == 0: return 'Fizz'
    if number%3 == 0: return 'Buzz'
    if number%5 == 0: return 'Fizz Buzz'
    return str(number)

def main():
    for i in range(1, 201):
        print(fizz_buzz(i))


if __name__ == '__main__':
    main()

File: 01-FizzBuzz/Answers/JavaScript/fizzbuzz2.js
// fizzbuzz2.js
// By kota ninomiya

/** 
 * fizzBuzz - fizzBuzzを判断して返す関数
 *
 *@param number {Number} 検証する値
 * @return {String} いずれの場合にも該当しない場合numberを返す
 */
function fizzBuzz(number){
    if(number%15 == 0) return 'FizzBuzz'
    if(number%3 == 0) return 'Fizz'
    if(number%5 == 0) return 'Buzz'
    return number
}

for(let i=1; i<=200; i++){
    console.log(fizzBuzz(i))
}